### PR TITLE
Picture 2.0 - Part 1

### PR DESCRIPTION
**Pull request by FinalAngel**
This is the first part of the picture improvements focusing on:

* Cleaned up file structure
* Removed Django <1.8 support
* Adapted ``README.txt``

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,47 +1,48 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-from setuptools import setup
+from setuptools import find_packages, setup
+
 from djangocms_picture import __version__
 
+
+REQUIREMENTS = [
+    'django-cms>=3.3.1',
```
Comment (FinalAngel) [inline setup.py]: will be changed to 3.2.0 in part 2

Comment (czpython) [inline setup.py]: shouldn't it be 3.3.0 in part two? if removing < django 1.8

Comment (FinalAngel) [inline setup.py]: 3.2 also supports 1.8+?

Comment (FinalAngel) [inline setup.py]: @czpython let me know if its okay to merge like that, can than continue adaptions in part 2 :)

**Comment by czpython** [comment]
Ok to merge 👍

**Pull request merged**